### ISSUE TITLE
Change Actor State

### ISSUE THREAD

**Issue by thiagob** [IndyJoeA/cbpi_BlynkPlugin#2]
Hi,

Do you think would be possible to change the actor states?

Mine are showing correctly but would be very good if I could turn on and turn off a pump for example...

**Comment by mjg964** [comment]
The front page says this functionality is planned for a later release.

**Comment by IndyJoeA** [comment]
Yes, this is at the top of my list of improvements for Blynk, however I have not been able to make it work with the current version of CBPi.  Hopefully with the new release of CBPi 3.1, plugins will be able to interact with other devices and plugins more easily and this should be possible.

**Comment by torrdan** [comment]
Hello, is there any update regarding the possibility to control the actors state, change setpoint? This would be very beneficial. 
I saw that Manuel is working on v 4: do you know if with that new release it would be possible? 
Thanks and congrats for the add-on.
